tools/licenses/lib/paths.dart

```diff
@@ -370,6 +370,7 @@ final Set<String> skippedCommonFiles = <String>{
   r'codereview.settings',
   r'coderules.txt',
   r'configure-ac-style.md',
+  r'deny.toml',
```

Comment:
I think we should probably just skip everything under the rust bindings directory under https://boringssl.googlesource.com/boringssl/+/refs/heads/master/rust since we don't build/ship them. That exclusion would go above next to the other paths we exclude from boringssl.